cmd/kubectl-gadget/utils/flags.go

```diff
@@ -191,19 +200,38 @@ func AddCommonFlags(command *cobra.Command, params *CommonFlags) {
 		fmt.Sprintf("Output format (%s).", strings.Join(commonutils.SupportedOutputModes, ", ")),
 	)
 
-	command.PersistentFlags().BoolVarP(
+	command.PersistentFlags().BoolVar(
 		&params.Verbose,
 		"verbose",
-		"",
 		false,
 		"Print additional information",
 	)
 
-	command.PersistentFlags().IntVarP(
+	command.PersistentFlags().IntVar(
 		&params.Timeout,
 		"timeout",
-		"",
 		0,
 		"Number of seconds that the gadget will run for",
 	)
+
+	command.PersistentFlags().StringVar(
+		&params.DockerSocketPath,
+		"docker-socketpath",
+		"",
+		"Docker Engine API Unix socket path",
+	)
+
+	command.PersistentFlags().StringVar(
+		&params.ContainerdSocketPath,
+		"containerd-socketpath",
+		"",
+		"containerd CRI Unix socket path",
+	)
+
+	command.PersistentFlags().StringVar(
+		&params.CrioSocketPath,
+		"crio-socketpath",
+		"",
+		"CRI-O CRI Unix socket path",
+	)
```

Comment:
I think you should move this to a dedicated function in a common place. Then, we can use it here, in the `deploy` command, and in `local-gadget`. Something similar to what we did for [the output flags](https://github.com/kinvolk/inspektor-gadget/blob/main/cmd/common/utils/flags.go). In this way, we can keep them aligned. WDYT?

Reply:
Seems good to me to move it to that place